packages/appstore/src/components/cfds-listing/cfds-listing.scss

```diff
@@ -1357,6 +1364,9 @@
                 width: unset;
                 height: unset;
             }
+            &__table-header {
+                width: calc(250vw / 7 * 3);
```

Comment:
Why is this calculated like this ?

Reply:
The other width is `250vw` which overflow the whole width so for exact width in the mobile view have to use this method. So the overflow-x is not stretched.